tox.ini

```diff
@@ -1,10 +1,17 @@
 [tox]
+skipsdist = True
```

Comment:
This will skip creating the fxapom distribution for testing, which is not what we want.